projects/msgpack-c/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -eu
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+cmake -DMSGPACK_CXX11=ON .
```

Comment:
CC, CXX, CXXFLAGS and CFLAGS are picked up by cmake from the environment right?

Reply:
Sorry I didn't have a preference for one or the other, just wanted to confirm that what you had initially worked. Thanks for looking into this